shared/connection.go

```diff
@@ -318,3 +397,32 @@ func chooseBackend[F interface{}](
 	// Should never happen if the commands are the same than those handled in GetCommand()
 	return nil, errors.New(L("no supported backend found"))
 }
+
+// extractNamespaceFromConfig extracts the namespace of a given application
+// from the Helm release information from an also given kubeconfig file (path).
```

Comment:
```suggestion
// from the Helm release information.
```
The kubeconfig path is just for the case it's installed in a non-standard place as k3s or rke2 are doing. If it's in `~/.kube/config` it can be empty.